plugins/html/icon_html/actions/epub/action.py

```diff
@@ -14,18 +14,17 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        temp_file = "temp_html_2_epub.epub"
+        temp_file = "temp_html_3_epub.epub"
         tag_parser = "(?i)<\/?\w+((\s+\w+(\s*=\s*(?:\".*?\"|'.*?'|[^'\">\s]+))?)+\s*|\s*)\/?>"  # noqa: W605
-        tags = re.findall(tag_parser, params.get("doc"))
-        try:
-            if not len(tags):
-                raise PluginException(cause="Run: Invalid input.", assistance="Input must be of type HTML.")
-            pypandoc.convert(params.get("doc"), "epub", outputfile=temp_file, format="html")
-            with open(temp_file, "rb") as output:
-                # Reading the output and sending it in base64
-                return {"epub": base64.b64encode(output.read()).decode("utf-8")}
-        except Exception:
-            return {"error": "Error occurred please try again"}
+        tags = re.findall(tag_parser, params.get(Input.DOC))
+
+        if not tags:
+            raise PluginException(cause="Run: Invalid input.", assistance="Input must be of type HTML.")
 
-    def test(self):
-        return {"test": "test Success"}
+        try:
+            pypandoc.convert(params.get(Input.DOC), "epub", outputfile=temp_file, format="html")
+        except RuntimeError as error:
+            raise PluginException(cause="Pypandoc Runtime Error: ", assistance="Check stack trace log", data=error)
```

Comment:
Similar suggestion to above re cause wording. (And all other exceptions raised with Pypandoc wording in the cause)